eng/config/PublishData.json

```diff
@@ -29,8 +29,8 @@
       ],
       "vsBranch": "main",
```

Comment:
Seems wrong. Should be the same as Roslyn, no?

Reply:
Roslyn's main is also inserting into VS main. If you are suggesting this should be feature/d18initial, I'm not sure about that, presumably after the snap, VS's main is the correct target branch for us to insert to.

Reply:
I'm not sure what you mean, this will be the same as Roslyn.

Reply:
Sorry, I was looking at Roslyn's earlier today and I thought it was set to the "initial" branch, but I guess that pre-snap anyway. I'm sure you know what you're doing better than me :)

Reply:
Yes, after VS snaps, I think main is the correct target branch rather than the "initial" one.